components/measurement/SummaryText.js

```diff
@@ -2,28 +2,45 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import moment from 'moment'
 import { Text } from 'ooni-components'
+import { FormattedMessage } from 'react-intl'
+
 import { getTestMetadata } from '../utils'
 
 const SummaryText = ({
   testName,
   network,
   country,
   date,
-  hint = 'is reachable',
+  content,
   testUrl,
 
 }) => {
   const metadata = getTestMetadata(testName)
   const formattedDate = moment(date).format('LL')
   const formattedDateTime = moment(date).format('lll')
-  return (
-    <Text py={4} fontSize={3}>
-      The <a href={metadata.info}>{metadata.name}</a> test
-      of {testUrl && <a href={testUrl}>{testUrl}</a>} in the {network} network
-      of {country} on <abbr title={formattedDateTime}>{formattedDate}</abbr>&nbsp;
-      {hint}. Check the network measurement data below, as well as other
-      measurements from this network testing this site to explore further.
+
+  let textToRender = null
+  if (typeof content === 'function') {
+    textToRender = content()
+  } else {
+    textToRender = <Text py={4} fontSize={3}>
+      <FormattedMessage id='Measurement.Details.SummaryTextTemplate'
+        defaultMessage='The {testName} test
+        in the {network} network of {country} on {date}
+        {content}. Check the network measurement data below, as well as other
+        measurements from this network testing this site to explore further.'
```

Comment:
Why does this have a defaultMessage and the others don't?

Is it because it is using tokens we need to have defaultMessage with tokens?

If that is the case to avoid confusion on where the translation comes from I suggest we put a default message that includes the token variables, but doesn't have meaningful text in it.

Maybe something like:
```
EDIT COPY IN CSV {testName} {network} {country}
```